scripts.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable no-plusplus */
```

Comment:
- [ ] Nice one adding the "featured speakers" section dynamically, Please don't disable linters, you can check online about the fixes, you can check stackoverflow and it likes. It's too early to start disabling linters 😁.